plugins/topic.py

```diff
@@ -75,7 +71,7 @@ async def music_of_the_day(self, msg):
         self.bot.log.info("Music of the day changed by " + msg.user)
         self.make_topic(msg, new_music=music_url)
         try:
-            title = await self.find_title(music_url)
+            title = self.find_title(music_url)
```

Comment:
Il faudrait soit utiliser une méthode async de youtube-dl, soit lancer ça dans un autre thread, sinon on va bloquer la mainloop le temps de faire la query

Reply:
Théoriquement oui,maintenant si la méthode prend "peu" de temps, je dirais qu'on s'en fout ?

Reply:
https://github.com/UrLab/lechbot/pull/64#discussion_r640730184
> And if it fails, it will return a bullshit string

C'est quoi cette string ? Pour bien faire, faudrait aussi remplacer le bullshit par "rien"

Reply:
Je dirai que soit la méthode prends peu de temps, soit elle ne marche pas tout court... Ça ne devrait pas prendre plus de 3 secondes

Reply:
Après une petite recherche, ça ne semble pas possible avec youtube-dl...